nwutils/nw-builder
# fix: `MacOS` icon and unofficial `MacOS` arm build

Fixes: #895

## Description

- Copy and overwrite `app.icns` with user defined icon if specified by user

## Testing steps

- Clone repo
- Checkout `dev-895` branch
- `npm i` to install node modules
- `npm run test:demo` to build MacOS app
- Generated app will be in `test/fixture/out` directory named `demo.app`
- Execute application via terminal or file explorer
- Verify that the NW Utils icon shows correctly as file icon and in the Dock

cc @rocketVibes
